src/client/cli/commands/publish.js

```diff
@@ -14,10 +14,11 @@ type HandlerOptions = {
   apiUrl: string,
   idSelector: string,
   contentSelector: ?string,
```

Comment:
no, I meant having the "?" moved from the type to the field, like the fellows below.
Not sure what it does in flow, just seemed forgotten.